hotel_reservation/__manifest__.py

```diff
@@ -13,6 +13,7 @@
     "demo": ["data/hotel_reservation_data.xml"],
     "data": [
         "security/ir.model.access.csv",
+        "security/reservation_security.xml",
```

Comment:
Add XML files first to render,  because CSV file will need the group existent before.

Reply:
done